Fix:
lexer.c:1955:40: error: implicit declaration of function 'isatty' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
https://github.com/codehelp/qof/issues/4
--- lib/libsql/lexer.c.orig	2014-09-23 02:40:17.000000000 -0500
+++ lib/libsql/lexer.c	2022-03-23 17:58:48.000000000 -0500
@@ -40,6 +40,7 @@
 #include <string.h>
 #include <errno.h>
 #include <stdlib.h>
+#include <unistd.h>
 
 /* end standard C headers. */
 
